grype/presenter/presenter.go

```diff
@@ -3,17 +3,14 @@ package presenter
 import (
 	"io"
 
-	cdx "github.com/CycloneDX/cyclonedx-go"
+	cyclonedxLib "github.com/CycloneDX/cyclonedx-go"
```

Comment:
nit: if we're refactoring this anyway, it might be nice to move this reference into the `github.com/anchore/grype/grype/presenter/cyclonedx` package and instead expose 2 functions from there: `cyclonedx. NewJSONPresenter` and `cyclonedx.NewXMLPresenter` or similar